bot/internal/review/review.go

```diff
@@ -237,23 +239,29 @@ func (r *Assignments) getAdminReviewers(author string) []string {
 	return reviewers
 }
 
-func (r *Assignments) getCodeReviewerSets(author string) ([]string, []string) {
+func (r *Assignments) getCodeReviewerSets(e *env.Environment) ([]string, []string) {
 	// Internal non-Core contributors get assigned from the admin reviewer set.
 	// Admins will review, triage, and re-assign.
-	v, ok := r.c.CodeReviewers[author]
-	if !ok || v.Team == "Internal" {
-		reviewers := r.getAdminReviewers(author)
+
+	v, ok := r.c.CodeReviewers[e.Author]
+	if !ok || v.Team == internalTeam {
+		reviewers := r.getAdminReviewers(e.Author)
 		n := len(reviewers) / 2
 		return reviewers[:n], reviewers[n:]
 	}
 
-	// Cloud gets reviewers assigned from Core.
 	team := v.Team
-	if v.Team == "Cloud" {
-		team = "Core"
+
+	// Cloud gets reviewers assigned from Core in Teleport
+	// Core gets reviewers assigned from Cloud in Cloud
```

Comment:
I thought I updated this to match the same comment below in `checkCodeReviews` but apparently not - late night coding...

```suggestion
	// Teams do their own internal reviews
```